### PR TITLE
Initial support for BigFix computers

### PR DESCRIPTION
**Pull request by achantavy**
Adds support for BigFix computers.

Sample:
image

Example invocation:
Screenshot 2023-04-19 at 12 13 40 PM

### CODE REVIEW

File: cartography/config.py [hunk 1]
```diff
@@ -91,6 +91,12 @@ class Config:
     :param lastpass_cid: Lastpass account ID. Optional.
     :type lastpass_provhash: str
     :param lastpass_provhash: Lastpass API KEY. Optional.
+    :type bigfix_username: str
+    :param bigfix_username: The username to authenticate to BigFix. Optional.
+    :type bigfix_password: str
+    :param bigfix_password: The password to authenticate to BigFix. Optional.
+    :type bigfix_root_url: str
+    :param bigfix_root_url: The API URL to use for BigFix, e.g. "https://example.com:52311". Optional.
```
Comment (ramonpetgrave64) [inline cartography/config.py:99]:
```suggestion
    :param bigfix_root_url: The API URL to use for BigFix, e.g. "https://example.com:52311".
```

File: cartography/intel/bigfix/computers.py [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    Returned shape: [
+      {'@Resource': 'https://{URI}/api/computer/123', 'LastReportTime': 'Tue, 18 Apr 2023 21:59:44 +0000', 'ID': '123'},
+    ]
+    """
+    xml_text = _get_computer_list_raw_xml(bigfix_api_url, headers)
+    as_dict = xmltodict.parse(xml_text)
+    return as_dict['BESAPI']['Computer']
+
+
+def get_computer_details(
```
Comment (ramonpetgrave64) [inline cartography/intel/bigfix/computers.py:118]: docstring

File: cartography/intel/bigfix/computers.py [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+    load_computers(neo4j_session, computers, bigfix_root_url, update_tag)
+    cleanup(neo4j_session, common_job_parameters)
+
+
+@timeit
+def get(bigfix_api_url: str, headers: Dict[str, str], computer_keys: Set[str]) -> List[Dict[str, Any]]:
+    result = []
+    computer_list = get_computer_list(bigfix_api_url, headers)
+    logger.info(f"Retrieving details for {len(computer_list)} BigFix Computers")
+    for computer in computer_list:
```
Comment (ramonpetgrave64) [inline cartography/intel/bigfix/computers.py:82]: put this in a _transform()

**Review by csanders-git — APPROVED** [review]

**Review by ramonpetgrave64 — APPROVED** [review]
non-blocking comments

**Pull request merged**